pkg/channel/event_receiver.go

```diff
@@ -256,12 +256,13 @@ func (r *EventReceiver) ServeHTTP(response nethttp.ResponseWriter, request *neth
 	features := feature.FromContext(ctx)
 	if features.IsOIDCAuthentication() {
 		r.logger.Debug("OIDC authentication is enabled")
-		err = r.tokenVerifier.VerifyJWTFromRequest(ctx, request, &r.audience, response)
+		oidcToken, err := r.tokenVerifier.GetOIDCIdentity(ctx, request, r.audience)
```

Comment:
The goal here is to verify the the request, I'd not change the existing code instead use a seperate function to get the OIDC Token!

Reply:
Okay , I'll add a separate function to do that and leave it as it was ..
Thanks !